### PR TITLE
Add reasonable warning when command on wheel extentions without --include-whl-extensions

### PR DESCRIPTION
**Pull request by haroldrandom**
* Optimize help message of linter
* Add another warning for command runing on a wheel installed extension without --include-whl-extensions
* Clean code
* Partially fix https://github.com/Azure/azure-cli/issues/11140
* Partially fix https://github.com/Azure/azure-cli-extensions/issues/1078

### CODE REVIEW

File: azdev/utilities/path.py [hunk 1]
```diff
@@ -208,40 +209,30 @@ def get_path_table(include_only=None, include_whl_extensions=False):
     modules_paths = glob(paths)
     core_paths = glob(os.path.normcase(os.path.join(cli_repo_path, 'src', '*', 'setup.py')))
     ext_paths = [x for x in find_files(ext_repo_paths, '*.*-info') if 'site-packages' not in x]
-    whl_ext_paths = []
-    if include_whl_extensions:
-        from azure.cli.core.extension import EXTENSIONS_DIR  # pylint: disable=import-error
-        whl_ext_paths = [x for x in find_files(EXTENSIONS_DIR, '*.*-info') if 'site-packages' not in x]
+    whl_ext_paths = [x for x in find_files(EXTENSIONS_DIR, '*.*-info') if 'site-packages' not in x]
```
Comment (MyronFanQiu) [inline azdev/utilities/path.py]: is it intentional to remove `if include_whl_extensions`? why?

**Comment by haroldrandom** [comment]
Partially fix https://github.com/Azure/azure-cli/issues/11140

File: azdev/utilities/path.py (hunk 1 above)
Comment (haroldrandom) [reply azdev/utilities/path.py]: Yes. it's intentional. `whl_ext_paths` only loaded when `--include-whl-extensions` is specified.
But user won't get a hint when entering a module name which installed from a wheel extension. 
So wheel-installed extensions should be loaded anyway such that azdev can find out whether the user-entering module name is a whell-installed extenions or not, such that can give user a more friendly hint of using `--include-whl-extenions`.

But this move has drawback now, cause only `linter` has supported this options.

**Pull request merged**